### ISSUE TITLE
Add. 14,541, foll. 11–18 | https://bl.vuexistapps.us/manuscript/1417 (Part2)

### ISSUE THREAD

**Issue by Erfan-Syriaca** [srophe/britishLibrary-data#931]
### Categories of Manuscript Overview Problems

- [ ] Whitespace
- [ ] Typo(s)
- [ ] Incorrect content
- [ ] Missing content

### Problems with Physical Description

- "Vellum leaves" perhaps should be changed to "Eight vellum leaves."
- The printed catalog reads: The quires are signed with letters (foll. 1a, 25a, 29a). I cannot clearly understand what it means. Did he mean that there are only three quires, and their numbers merely appear on those three folios? If this is correct, no quire number appears in this portion of the manuscript (foll. 11-18). The sentence mentioned above has been inserted in the Phys. Des. of this manuscript's entry. So, please check if it should be there. 
- The word "Date" should perhaps be deleted in "These leaves are taken from a manuscript of the Date:ixth cent."

### Categories of Physical Description Problems

- [ ] Whitespace
- [X] Typo(s)
- [X] Incorrect content
- [X] Missing content

**Comment by mikey-tang16** [comment]
- "Vellum leaves" changed to "Eight vellum leaves."

Question
- The sentence about "the quires signed.." seems correct to me. The corresponding numbers appear correctly in the separated entries.
@wlpotter

**Comment by wlpotter** [comment]
Yes that seems right to me. I've opened #1969 to track adding an editorial note, and an issue with the parts (we should have just one part for foll. 19-24, 27, 28 but split it into two). That's a pretty in depth change, so I'll take care of that one. Feel free to close this issue if there's nothing else that needs to be updated

**Comment by wlpotter** [comment]
Updated the part divisions
